app/services/start_price_thresholds.rb

```diff
@@ -0,0 +1,8 @@
+class StartPriceThresholds < Struct.new(:start_price)
+  MICROPURCHASE = 3500
+  SAT = 150000
+
+  def small_business?
```

Comment:
seems like whether an auction is for small businesses is a cncern of eligibility more than start price threshold